dml_events.go

```diff
@@ -253,84 +250,77 @@ func NewBinlogDMLEvents(table *schema.Table, ev *replication.BinlogEvent, pos my
 	}
 }
 
-func loadColumnsForTable(table *schema.Table, valuesToVerify ...RowData) ([]string, error) {
-	for _, values := range valuesToVerify {
-		if err := verifyValuesHasTheSameLengthAsColumns(table, values); err != nil {
```

Comment:
This method is now instead called manually in the `AsSQLQuery` methods directly.